### PR TITLE
feat(explorer): wider experiment result view modal + more download types

### PR DESCRIPTION
**Pull request by noctillion**
Expand support for documents data results download and vizualization

### CODE REVIEW

File: src/components/explorer/IndividualExperiments.js [hunk 1]
```diff
@@ -57,7 +57,11 @@ const ExperimentResultActions = ({ result }) => {
 
     return <>
         {resultViewable ? <>
-            <Modal title={<span>View: {result.filename}</span>} visible={viewModalVisible} onCancel={onViewCancel}>
+            <Modal
+                title={<span>View: {result.filename}</span>}
+                visible={viewModalVisible}
+                onCancel={onViewCancel}
+                width={695}>
```
Comment (davidlougheed) [inline src/components/explorer/IndividualExperiments.js:64]: maybe use same value as drop box for modal width? (1080)

File: src/components/explorer/IndividualExperiments.js [hunk 2]
```diff
@@ -353,7 +357,25 @@ const IndividualExperiments = ({ individual }) => {
 
 // expand here accordingly
 const isDownloadable = (result) =>
```
Comment (davidlougheed) [inline src/components/explorer/IndividualExperiments.js:359]: is there a reason some files aren't downloadable?

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/propTypes.js [hunk 3]
```diff
@@ -312,26 +312,13 @@ export const experimentResultPropTypesShape = PropTypes.shape({
     description: PropTypes.string,
     filename: PropTypes.string,
     file_format: PropTypes.oneOf([
-        "SAM",
```
Comment (davidlougheed) [inline src/propTypes.js:315]: this change is incorrect - it should match the schema in katsu

File: src/components/explorer/IndividualExperiments.js [hunk 4]
```diff
@@ -352,8 +361,27 @@ const IndividualExperiments = ({ individual }) => {
 };
 
 // expand here accordingly
-const isDownloadable = (result) =>
-    ["vcf", "cram", "bw", "bigwig"].includes(result.file_format?.toLowerCase());
+const isDownloadable = (result) => {
```
Comment (davidlougheed) [inline src/components/explorer/IndividualExperiments.js:364]: just remove the isDownloadable function/check - if the file exists it should be downloadable

File: src/propTypes.js (hunk 3 above)
Comment (noctillion) [reply src/propTypes.js:315]: Thanks for the correction!

File: src/components/explorer/IndividualExperiments.js [hunk 5]
```diff
@@ -274,7 +274,7 @@ const Experiments = ({ individual, handleExperimentClick }) => {
             .flatMap((e) => e?.experiment_results ?? [])
             .map((r) => ({  // enforce file_format property
                 ...r,
-                file_format: r.file_format ? r.file_format.toLowerCase() : guessFileType(r.filename).toLowerCase(),
+                file_format: r.file_format ? r.file_format.toLowerCase() : guessFileType(r.filename),
```
Comment (davidlougheed) [inline src/components/explorer/IndividualExperiments.js:277]: change back to `r.file_format ?? guessFileType(r.filename),`

**Review by davidlougheed — CHANGES_REQUESTED** [review]

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**